packages/socket-stream-client/node.js

```diff
@@ -107,10 +107,21 @@ export class ClientStream extends StreamClientCommon {
   _getProxyUrl(targetUrl) {
     // Similar to code in tools/http-helpers.js.
     var proxy = process.env.HTTP_PROXY || process.env.http_proxy || null;
+    var noproxy = process.env.NO_PROXY || process.env.no_proxy || null;
     // if we're going to a secure url, try the https_proxy env variable first.
-    if (targetUrl.match(/^wss:/)) {
+    if (targetUrl.match(/^wss:/) || targetUrl.match(/^https:/)) {
       proxy = process.env.HTTPS_PROXY || process.env.https_proxy || proxy;
     }
+    if (targetUrl.indexOf('localhost') != -1 || targetUrl.indexOf('127.0.0.1') != -1) {
+      return null;
+    }
+    if (noproxy) {
+      for (let item of noproxy.split(',')) {
+        if (targetUrl.indexOf(item.trim().replace(/\*/, '')) !== -1) {
+          return null;
```

Comment:
Hello,
You should replace this line by: `proxy = null`
We have just tested with this correction and it works.
Hope this will help.
Bye